messages/en.json

```
@@ -7,6 +7,7 @@
   "admin/b2b-organizations.selected-rows": "Selected rows: {qty}",
   "admin/b2b-organizations.showRows": "Show rows",
   "admin/b2b-organizations.of": "of",
+  "admin/b2b-organizations.invalid-schema": "The CL schema is invalid, please see the b2b suite documentation (Frequently Asked Questions) for more information",
```

```suggestion
  "admin/b2b-organizations.invalid-schema": "B2B user creation is currently disabled because the MasterData CL (user profile) schema is invalid on this account. Please see the B2B Suite documentation (Frequently Asked Questions) for more information.",
```
Let's also include a link to https://developers.vtex.com/vtex-developer-docs/docs/vtex-b2b-suite please